From 3b4c7e3119910cce97efb0c112ec33e7e7a0d284 Mon Sep 17 00:00:00 2001
From: flyranchao <flyranchao@allwinnertech.com>
Date: Fri, 12 Jul 2019 20:00:43 +0800
Subject: [PATCH] r7s: update some config

	1.config-4.9:
		add standby debug config.
		cancle scenc_lock config.
	2.defconfig:
		cancle rtc tinatest case.
		add cpucle tinatest case.
		add standby tinatest case.
		add benchmarks tinatest case.

Change-Id: I66b55d308883feeaa8d5d7f59c5e8bfa58858f2d
Signed-off-by: flyranchao <flyranchao@allwinnertech.com>
---
 allwinner/r7s-R7s/config-4.9 |   7 +-
 allwinner/r7s-R7s/defconfig  | 165 +++++++++++++++++++++++++++++++++---------
 2 files changed, 134 insertions(+), 38 deletions(-)

diff --git a/allwinner/r7s-R7s/config-4.9 b/allwinner/r7s-R7s/config-4.9
index 5d3ce65..d181da8 100644
--- a/allwinner/r7s-R7s/config-4.9
+++ b/allwinner/r7s-R7s/config-4.9
@@ -608,9 +608,11 @@ CONFIG_PINCTRL_SUNXI=y
 # CONFIG_PL310_ERRATA_753970 is not set
 # CONFIG_PL310_ERRATA_769419 is not set
 CONFIG_PM=y
+CONFIG_PM_ADVANCED_DEBUG=y
 CONFIG_PM_CLK=y
-# CONFIG_PM_DEBUG is not set
+CONFIG_PM_DEBUG=y
 CONFIG_PM_SLEEP=y
+CONFIG_PM_SLEEP_DEBUG=y
 CONFIG_POWER_SUPPLY=y
 CONFIG_PREEMPT=y
 # CONFIG_PREEMPTIRQ_EVENTS is not set
@@ -680,7 +682,7 @@ CONFIG_RTL8189ES=m
 CONFIG_RT_GROUP_SCHED=y
 CONFIG_RWSEM_XCHGADD_ALGORITHM=y
 CONFIG_SBITMAP=y
-CONFIG_SCENELOCK=y
+# CONFIG_SCENELOCK is not set
 CONFIG_SCHEDSTATS=y
 CONFIG_SCHED_AUTOGROUP=y
 CONFIG_SCHED_HRTICK=y
@@ -906,7 +908,6 @@ CONFIG_USB_SUPPORT=y
 CONFIG_USB_UAS=y
 # CONFIG_USERIO is not set
 # CONFIG_USER_NS is not set
-CONFIG_USER_SCENELOCK=y
 CONFIG_USE_OF=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
 CONFIG_VDSO=y
diff --git a/allwinner/r7s-R7s/defconfig b/allwinner/r7s-R7s/defconfig
index fa33f14..c94949e 100644
--- a/allwinner/r7s-R7s/defconfig
+++ b/allwinner/r7s-R7s/defconfig
@@ -34,14 +34,13 @@ CONFIG_HAVE_DOT_CONFIG=y
 # CONFIG_TARGET_cello_pro is not set
 # CONFIG_TARGET_cowbell_demo is not set
 # CONFIG_TARGET_cowbell_perf1 is not set
+# CONFIG_TARGET_cowbell_perf1_rt is not set
 # CONFIG_TARGET_cowbell_perf2_xr829 is not set
 # CONFIG_TARGET_cowbell_qa is not set
 # CONFIG_TARGET_cowbell_std is not set
 # CONFIG_TARGET_cowbell_ubidemo is not set
 # CONFIG_TARGET_drum_std is not set
 # CONFIG_TARGET_drum_std_xr829 is not set
-# CONFIG_TARGET_eyesee_dvb is not set
-# CONFIG_TARGET_eyesee_sdv is not set
 # CONFIG_TARGET_fr7s_ac100 is not set
 # CONFIG_TARGET_koto_carp is not set
 # CONFIG_TARGET_koto_perf1 is not set
@@ -53,6 +52,7 @@ CONFIG_TARGET_r7s_R7s=y
 # CONFIG_TARGET_octopus_dev is not set
 # CONFIG_TARGET_octopus_sch is not set
 # CONFIG_TARGET_organ_perf1 is not set
+# CONFIG_TARGET_petrel_p1 is not set
 # CONFIG_TARGET_piano_chimera is not set
 # CONFIG_TARGET_piano_p1 is not set
 # CONFIG_TARGET_piano_perf1 is not set
@@ -74,9 +74,12 @@ CONFIG_TARGET_r7s_R7s=y
 # CONFIG_TARGET_tulip_perf1 is not set
 # CONFIG_TARGET_tulip_pine64 is not set
 # CONFIG_TARGET_tulip_sgw is not set
+# CONFIG_TARGET_v313_sdv is not set
 # CONFIG_TARGET_v316_SdvDragonBoard is not set
 # CONFIG_TARGET_v316_perfnor is not set
 # CONFIG_TARGET_v316_sdv is not set
+# CONFIG_TARGET_v5_dvb is not set
+# CONFIG_TARGET_v5_lindeni is not set
 # CONFIG_TARGET_v536_CdrDragonboard is not set
 # CONFIG_TARGET_v536_cdr is not set
 # CONFIG_TARGET_violin_F1C200s is not set
@@ -1504,7 +1507,16 @@ CONFIG_PACKAGE_adb=y
 CONFIG_PACKAGE_alsa-conf-aw=y
 # CONFIG_PACKAGE_alsa-plugins-aw is not set
 # CONFIG_PACKAGE_asan_demo is not set
-# CONFIG_PACKAGE_benchmarks is not set
+# CONFIG_PACKAGE_awsystrace is not set
+CONFIG_PACKAGE_benchmarks=y
+
+#
+# Configuration
+#
+# CONFIG_BENCHMARKS_DHRYSTONE is not set
+CONFIG_BENCHMARKS_LINPACK=y
+CONFIG_BENCHMARKS_COREMARK=y
+# CONFIG_BENCHMARKS_STATIC_COMPILE is not set
 # CONFIG_PACKAGE_boot-play is not set
 CONFIG_PACKAGE_camerademo=y
 # CONFIG_ENABLE_VIN_ISP is not set
@@ -1605,6 +1617,7 @@ CONFIG_JPEGDECODER=y
 CONFIG_TR_NO_USE_VIN_ISP=y
 # CONFIG_CEDARX_YES_USE_PLAYREADY is not set
 CONFIG_CEDARX_NO_USE_PLAYREADY=y
+# CONFIG_PACKAGE_libdisplay is not set
 # CONFIG_PACKAGE_libduersdk is not set
 # CONFIG_PACKAGE_libminui is not set
@@ -1622,6 +1635,7 @@ CONFIG_SUNXI_DISPLAY_DE=y
 # CONFIG_PACKAGE_logwrapper is not set
 # CONFIG_PACKAGE_mad-demo is not set
 # CONFIG_PACKAGE_misc-upgrade is not set
+# CONFIG_PACKAGE_mtdbadblk is not set
 CONFIG_PACKAGE_mtop=y
 # CONFIG_PACKAGE_nativepower is not set
 CONFIG_PACKAGE_nativepower_utils=y
@@ -1639,11 +1653,11 @@ CONFIG_PACKAGE_softap-demo=y
 CONFIG_REALTEK=y
 # CONFIG_ESP8089 is not set
-# CONFIG_PACKAGE_stress-ng is not set
+CONFIG_PACKAGE_stress-ng=y
 CONFIG_PACKAGE_swupdate=y
 CONFIG_SWUPDATE_CUSTOM=y
 CONFIG_SWUPDATE_DEFAULT_HAVE_DOT_CONFIG=y
-CONFIG_SWUPDATE_DEFAULT_CURL=y
+# CONFIG_SWUPDATE_DEFAULT_CURL is not set
 # CONFIG_SWUPDATE_DEFAULT_CURL_SSL is not set
 # CONFIG_SWUPDATE_DEFAULT_SYSTEMD is not set
 CONFIG_SWUPDATE_DEFAULT_SCRIPTS=y
@@ -1652,7 +1666,7 @@ CONFIG_SWUPDATE_DEFAULT_SW_VERSIONS_FILE="/etc/sw-versions"
 CONFIG_SWUPDATE_DEFAULT_SOCKET_CTRL_PATH="/tmp/sockinstctrl"
 CONFIG_SWUPDATE_DEFAULT_SOCKET_PROGRESS_PATH="/tmp/swupdateprog"
 CONFIG_SWUPDATE_DEFAULT_SOCKET_REMOTE_HANDLER_DIRECTORY="/tmp/"
-CONFIG_SWUPDATE_DEFAULT_MTD=y
+# CONFIG_SWUPDATE_DEFAULT_MTD is not set
 # CONFIG_SWUPDATE_DEFAULT_LUA is not set
 # CONFIG_SWUPDATE_DEFAULT_FEATURE_SYSLOG is not set
 CONFIG_SWUPDATE_DEFAULT_CROSS_COMPILE=""
@@ -1668,9 +1682,9 @@ CONFIG_SWUPDATE_DEFAULT_UBOOT=y
 # CONFIG_SWUPDATE_DEFAULT_BOOTLOADER_NONE is not set
 # CONFIG_SWUPDATE_DEFAULT_BOOTLOADER_GRUB is not set
 CONFIG_SWUPDATE_DEFAULT_UBOOT_FWENV="/etc/fw_env.config"
-CONFIG_SWUPDATE_DEFAULT_DOWNLOAD=y
+# CONFIG_SWUPDATE_DEFAULT_DOWNLOAD is not set
 # CONFIG_SWUPDATE_DEFAULT_DOWNLOAD_SSL is not set
-CONFIG_SWUPDATE_DEFAULT_CHANNEL_CURL=y
+# CONFIG_SWUPDATE_DEFAULT_CHANNEL_CURL is not set
 # CONFIG_SWUPDATE_DEFAULT_HASH_VERIFY is not set
 # CONFIG_SWUPDATE_DEFAULT_SIGNED_IMAGES is not set
 # CONFIG_SWUPDATE_DEFAULT_ENCRYPTED_IMAGES is not set
@@ -1787,6 +1801,13 @@ CONFIG_SWUPDATE_CONFIG_SHELLSCRIPTHANDLER=y
 CONFIG_SWUPDATE_CONFIG_BOOTLOADERHANDLER=y
 CONFIG_SWUPDATE_CONFIG_AWBOOT_HANDLER=y
 # CONFIG_SWUPDATE_CONFIG_UCFWHANDLER is not set
+
+#
+# Reduce size
+#
+# CONFIG_SWUPDATE_CONFIG_REMOVE_CLIENT is not set
+# CONFIG_SWUPDATE_CONFIG_REMOVE_PROGRESS is not set
+# CONFIG_SWUPDATE_CONFIG_REMOVE_AUTORUN is not set
 # CONFIG_PACKAGE_tconfigs is not set
 # CONFIG_PACKAGE_tina-app-upgrade is not set
 # CONFIG_PACKAGE_tina-upgrade is not set
@@ -2719,8 +2740,6 @@ CONFIG_ALSA_LIB_EXTRA_ucm=y
 # CONFIG_PACKAGE_libavahi-nodbus-support is not set
 # CONFIG_PACKAGE_libb64 is not set
 # CONFIG_PACKAGE_libbfd is not set
-# CONFIG_PACKAGE_libblkid is not set
-CONFIG_PACKAGE_libblobmsg-json=y
 # CONFIG_PACKAGE_libcap is not set
 # CONFIG_PACKAGE_libcares is not set
 # CONFIG_PACKAGE_libcharset is not set
@@ -2773,11 +2792,13 @@ CONFIG_LIBCURL_PROXY=y
 # CONFIG_LIBCURL_LIBCURL_OPTION is not set
 # CONFIG_LIBCURL_VERBOSE is not set
 # CONFIG_PACKAGE_libdaemon is not set
-# CONFIG_PACKAGE_libdrm is not set
-# CONFIG_PACKAGE_libelf1 is not set
 # CONFIG_PACKAGE_libdb47 is not set
 # CONFIG_PACKAGE_libdb47xx is not set
 # CONFIG_PACKAGE_libdbus is not set
+# CONFIG_PACKAGE_libdrm is not set
+# CONFIG_PACKAGE_libelf1 is not set
+# CONFIG_PACKAGE_libblkid is not set
+CONFIG_PACKAGE_libblobmsg-json=y
 # CONFIG_PACKAGE_libdevmapper is not set
 # CONFIG_PACKAGE_libepoxy is not set
 # CONFIG_PACKAGE_libev is not set
@@ -2872,6 +2893,7 @@ CONFIG_PACKAGE_libnl-tiny=y
 # CONFIG_PACKAGE_librpc is not set
 # CONFIG_PACKAGE_librrd1 is not set
 # CONFIG_PACKAGE_librsvg is not set
+# CONFIG_PACKAGE_librsync is not set
 # CONFIG_PACKAGE_librtlsdr is not set
 # CONFIG_PACKAGE_libruby is not set
 # CONFIG_PACKAGE_libsamplerate is not set
@@ -3831,22 +3853,7 @@ CONFIG_TINATEST_BASE_PRODUCTION_KEYTESTER_NUMBER_OF_KEYS=2
 # Advanced Customized
 #
 # CONFIG_TINATEST_BASE_PRODUCTION_KEYTESTER_ADVANCED is not set
-CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_ENABLE=y
-CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_COMMAND="rtctester.sh"
-CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_STDIN=""
-CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_FSTDIN=""
-CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_DATE=y
-CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_RESOURCE=y
-CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_RUN_TIMES=1
-CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_TESTCASE_RUN_ONCE_TIME=""
-CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_TESTCASE_RUN_TIME=""
-# CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_TIMEOUT_AS_FAILED is not set
-# CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_EXIT_ONCE_FAILED is not set
-
-#
-# Advanced Customized
-#
-# CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_ADVANCED is not set
+# CONFIG_TINATEST_BASE_PRODUCTION_RTCTESTER_ENABLE is not set
 CONFIG_TINATEST_BASE_PRODUCTION_EMMCTESTER_ENABLE=y
 CONFIG_TINATEST_BASE_PRODUCTION_EMMCTESTER_COMMAND="emmctester.sh"
 CONFIG_TINATEST_BASE_PRODUCTION_EMMCTESTER_STDIN=""
@@ -4058,7 +4065,53 @@ CONFIG_TINATEST_BASE_AUDIO_MIC_CAPTURE_PLAYBACK_PCM_DEVICE="PlaybackHeadphone"
 # CONFIG_TINATEST_BASE_AUDIO_MIC_CAPTURE_ADVANCED is not set
 # CONFIG_TINATEST_BASE_DISPLAY is not set
 # CONFIG_TINATEST_BASE_WIFI is not set
-# CONFIG_TINATEST_SPEC is not set
+CONFIG_TINATEST_SPEC=y
+CONFIG_TINATEST_SPEC_BENCHMARKS=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_ENABLE=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_COMMAND="coremark"
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_STDIN=""
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_FSTDIN=""
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_DATE=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_RESOURCE=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_REAL_TIME_LOG=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_RUN_TIMES=1
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_RUN_ALONE=y
+# CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_RUN_PARALLEL is not set
+# CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_MAY_REBOOT is not set
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_TESTCASE_RUN_ONCE_TIME="30"
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_TESTCASE_RUN_TIME=""
+# CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_TIMEOUT_AS_FAILED is not set
+# CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_EXIT_ONCE_FAILED is not set
+
+#
+# Advanced Customized
+#
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_ADVANCED=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_INFO=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_COREMARK_LIMIT=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_ENABLE=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_COMMAND="linpack 200"
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_STDIN=""
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_FSTDIN=""
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_DATE=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_RESOURCE=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_REAL_TIME_LOG=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_RUN_TIMES=1
+# CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_RUN_ALONE is not set
+# CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_RUN_PARALLEL is not set
+# CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_MAY_REBOOT is not set
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_TESTCASE_RUN_ONCE_TIME="30"
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_TESTCASE_RUN_TIME=""
+# CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_TIMEOUT_AS_FAILED is not set
+# CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_EXIT_ONCE_FAILED is not set
+
+#
+# Advanced Customized
+#
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_ADVANCED=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_INFO=y
+CONFIG_TINATEST_SPEC_BENCHMARKS_LINPACK_LIMIT=y
+# CONFIG_TINATEST_SPEC_STORAGE is not set
 CONFIG_TINATEST_STRESS=y
 # CONFIG_TINATEST_STRESS_REBOOT_ENABLE is not set
 # CONFIG_TINATEST_STRESS_GPU_ENABLE is not set
@@ -4088,25 +4141,67 @@ CONFIG_TINATEST_STRESS_DDR_LOOP_TIMES=1000
 CONFIG_TINATEST_STRESS_DDR_ADVANCED=y
 CONFIG_TINATEST_STRESS_DDR_INFO=y
 CONFIG_TINATEST_STRESS_DDR_LIMIT=y
-# CONFIG_TINATEST_STRESS_CPU is not set
+CONFIG_TINATEST_STRESS_CPU=y
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_ENABLE=y
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_COMMAND="cpucalc.sh all"
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_STDIN=""
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_FSTDIN=""
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_DATE=y
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_RESOURCE=y
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_REAL_TIME_LOG=y
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_RUN_TIMES=1
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_RUN_ALONE=y
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_TESTCASE_RUN_ONCE_TIME="0 0 8"
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_TESTCASE_RUN_TIME=""
+# CONFIG_TINATEST_STRESS_CPU_CPUCALC_TIMEOUT_AS_FAILED is not set
+# CONFIG_TINATEST_STRESS_CPU_CPUCALC_EXIT_ONCE_FAILED is not set
+
+#
+# Advanced Customized
+#
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_ADVANCED=y
+CONFIG_TINATEST_STRESS_CPU_CPUCALC_INFO=y
+# CONFIG_TINATEST_STRESS_CPU_CPUCALC_LIMIT is not set
+# CONFIG_TINATEST_STRESS_CPU_CPUFREQ_ENABLE is not set
+# CONFIG_TINATEST_STRESS_CPU_CPUHOTPLUG_ENABLE is not set
 # CONFIG_TINATEST_STRESS_STORAGE is not set
 # CONFIG_TINATEST_STRESS_WIFI_ENABLE is not set
-# CONFIG_TINATEST_STRESS_STANDBY_ENABLE is not set
+CONFIG_TINATEST_STRESS_STANDBY_ENABLE=y
+CONFIG_TINATEST_STRESS_STANDBY_COMMAND="standby.sh"
+CONFIG_TINATEST_STRESS_STANDBY_STDIN=""
+CONFIG_TINATEST_STRESS_STANDBY_FSTDIN=""
+CONFIG_TINATEST_STRESS_STANDBY_DATE=y
+CONFIG_TINATEST_STRESS_STANDBY_RESOURCE=y
+CONFIG_TINATEST_STRESS_STANDBY_REAL_TIME_LOG=y
+CONFIG_TINATEST_STRESS_STANDBY_RUN_TIMES=1
+CONFIG_TINATEST_STRESS_STANDBY_RUN_ALONE=y
+# CONFIG_TINATEST_STRESS_STANDBY_RUN_PARALLEL is not set
+CONFIG_TINATEST_STRESS_STANDBY_MAY_REBOOT=y
+CONFIG_TINATEST_STRESS_STANDBY_TESTCASE_RUN_ONCE_TIME=""
+CONFIG_TINATEST_STRESS_STANDBY_TESTCASE_RUN_TIME=""
+# CONFIG_TINATEST_STRESS_STANDBY_TIMEOUT_AS_FAILED is not set
+# CONFIG_TINATEST_STRESS_STANDBY_EXIT_ONCE_FAILED is not set
+CONFIG_TINATEST_STRESS_STANDBY_TEST_NUMBERS=5
+CONFIG_TINATEST_STRESS_STANDBY_STANDBY_PERIOD=5
+CONFIG_TINATEST_STRESS_STANDBY_RUNNING_PERIOD=5
+
+#
+# Advanced Customized
+#
+CONFIG_TINATEST_STRESS_STANDBY_ADVANCED=y
+CONFIG_TINATEST_STRESS_STANDBY_INFO=y
+CONFIG_TINATEST_STRESS_STANDBY_LIMIT=y
 
 #
 # Thirdparty
 #
 # CONFIG_PACKAGE_duilite-app is not set
 # CONFIG_PACKAGE_duilite-lib is not set
-CONFIG_DUILITE_R328=y
 # CONFIG_PACKAGE_rtk_hcidump is not set
 # CONFIG_PACKAGE_tutuclear-demo is not set
 # CONFIG_PACKAGE_tutuclear-lib is not set
 # CONFIG_PACKAGE_uvoice-demo is not set
 # CONFIG_PACKAGE_uvoice-lib is not set
-CONFIG_UVOICE_AC_XIAOZHI_R328=y
-# CONFIG_UVOICE_HR is not set
-# CONFIG_UVOICE_YM is not set
 
 #
 # Utilities
-- 
1.9.1

